packages/docs/package.json

```diff
@@ -1,5 +1,12 @@
 {
   "name": "@lest/docs",
   "private": true,
-  "description": "API documentation in JSON form."
+  "description": "API documentation in JSON form.",
+  "main": "dist/index.js",
+  "type": "commonjs",
+  "scripts": {
+    "build": "tsc",
+    "preinstall": "npm run build",
```

Comment:
Is this still required with the top level tsconfig? 

I have not had to do this in any TS monorepo...

Reply:
This appears to be standard for any TS package. it's just that most will use `prepack` and only send JS files to npm, but since this is a local package, it builds for each install.

Reply:
It was not required